packages/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx

```diff
@@ -31,24 +22,21 @@ const iconMap: (outOfOffice: boolean) => Record<PresenceBadgeStatus, JSX.Element
 /**
  * Returns the props and state required to render the component
  */
-export const usePresenceBadge = (
-  props: PresenceBadgeProps,
-  ref: React.Ref<HTMLElement>,
-  defaultProps?: PresenceBadgeProps,
-): PresenceBadgeState => {
-  const state = useBadge(
-    props,
-    ref,
-    mergeProps(
+export const usePresenceBadge = (props: PresenceBadgeProps, ref: React.Ref<HTMLElement>): PresenceBadgeState => {
+  const state: PresenceBadgeState = {
+    ...useBadge(
       {
         size: 'small',
-        status: 'available',
-        outOfOffice: false,
-        icon: { as: 'span' },
+        ...props,
```

Comment:
Also here

Reply:
This use of `...props` is actually correct as-is, because it's being passed as the props argument to `useBadge`